app/controllers/redirect_to_accept_terms_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RedirectToAcceptTermsController < ApplicationController
+	def show
```

Comment:
This should be the #edit action instead of #show, since it is rendering an edit form.